vault/eventbus/bus.go

```diff
@@ -50,11 +61,30 @@ func (bus *EventBus) Start() {
 // Send sends an event to the event bus and routes it to all relevant subscribers.
 // This function does *not* wait for all subscribers to acknowledge before returning.
 func (bus *EventBus) Send(ctx context.Context, eventType logical.EventType, data *logical.EventData) error {
+	var nspace string
+	ns, err := namespace.FromContext(ctx)
+	if err == namespace.ErrNoNamespace {
+	} else if err != nil {
+		return err
+	} else {
+		nspace = ns.ID
+	}
 	if !bus.started.Load() {
 		return ErrNotStarted
 	}
+	pluginInfo, ok := ctx.Value(contextEventPluginInfo).(*logical.EventPluginInfo)
+	if !ok {
+		pluginInfo = nil
+	}
+
 	bus.logger.Info("Sending event", "event", data)
```

Comment:
Any reason to log `data` instead of `eventReceived`? I think we might end up changing the level here to debug, as it seems likely people won't pay much attention unless they're debugging a specific issue to do with missed events, but I'm also fine with leaving it as info and awaiting feedback as it gives a nice boost to visibility.

Reply:
Good call on the eventReceived. I'll keep it at INFO for now? But you're right in that it is a bit chatty probably.